crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__ruff_test__statement__with_py.snap

```diff
@@ -0,0 +1,123 @@
+---
+source: crates/ruff_python_formatter/src/lib.rs
+expression: snapshot
+---
+## Input
+```py
+with aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa, aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa:
+    ...
+    # trailing
+
+with a, a:  # after colon
+    ...
+    # trailing
+
+with (
+    aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
+    aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,
+):
+    ...
+    # trailing
+
+
+with call(arg):
+    call(arg)
+# fmt: skip
```

Comment:
Is this `fmt: skip` intentional?

Reply:
oops, this whole block is leftover from debugging. will remote